apollo-android-support/src/main/java/com/apollographql/apollo/ApolloCallback.java

```diff
@@ -52,10 +52,10 @@ public ApolloCallback(@Nonnull ApolloCall.Callback<T> callback, @Nonnull Handler
     });
   }
 
-  @Override public void onCompleted() {
```

Comment:
Is on completed not a thing anymore?

Reply:
it's now part of `onStatusEvent`, you got notification with `StatusEvent.COMPLETED`